scripts/metadata/generate_metadata.R

```diff
@@ -1418,7 +1418,7 @@ if(nrow(right) > 0){
   individuals$efficacy_absent_most_recent_visit <- 0
 }
 # efficacy_most_recent_visit_present
-right <- 
```

Comment:
Is there anyway to modify the settings in your code editor so the next time we have a PR it doesn't get filled with all the non-change changes?